docs/docs/noir/tooling/debugger.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Debugger
+description: Learn about the Noir Debugger, in its REPL or VS Code versions.
+keywords: [Nargo, VSCode, Visual Studio Code, REPL, Debugger]
+sidebar_position: 2
+---
+
+# Noir Debugger
+
+There are currently two ways of debugging Noir programs:
+
+1. From VS Code, via the [vscode-noir](https://github.com/noir-lang/vscode-noir) extension. You can install it via the [Visual Studio Marketplace](https://marketplace.visualstudio.com/items?itemName=noir-lang.vscode-noir).
+2. Via the REPL debugger, which ships with Nargo.
+
+In order to use either version of the debugger, you will need to install recent enough versions of Noir, [Nargo](../../getting_started/installation) and vscode-noir:
+
+- Noir 0.xx 
+- Nargo 0.xx
+- vscode-noir 0.xx
+
+:::info
+At the moment, the debugger supports debugging binary projects, but not contracts.
+:::
+
+## VS Code debugger quickstart
+
+Once you installed Nargo and the vscode-noir extension, you can start debugging your Noir programs by simply opening a `.nr` file, clicking the debugger pane, and clicking _Run and debug_. Alternatively you can just use the `F5` keyboard shortcut.
+
+You should be seeing something like this:
+
+![Screencast of VS Code Noir Debugger](@site/static/img/debugger/debugger-intro.gif)
+)
+
+We cover the VS Code Noir debugger more in depth in [its how-to guide](../../how_to/debugger/debugging_with_vs_code.md) and [its reference](../../reference/debugger/debugger_vscode.md).
+
+## REPL debugger quickstart
```

Comment:
Yeah I wouldn't put a quickstart here, feel free to simply mention that Nargo ships with the `debug` command (point to reference) and link to the how-to-guide.

Reason is mostly to be aligned with our docs framework Diátaxis and reducing maintainability in when changes come to the debugger

Reply:
Done!